llvm/test/MC/AMDGPU/gfx12_asm_ds_alias.s

```diff
@@ -23,3 +23,9 @@ ds_min_rtn_f32 v5, v1, v2
 
 ds_min_rtn_f64 v[5:6], v1, v[2:3]
 // GFX12: [0x00,0x00,0xc8,0xd9,0x01,0x02,0x00,0x05]
+
+ds_subrev_u32 v1, v2
+// GFX12: [0x00,0x00,0x08,0xd8,0x01,0x02,0x00,0x00]
+
+ds_subrev_u64 v1, v[2:3]
+// GFX12: [0x00,0x00,0x08,0xd9,0x01,0x02,0x00,0x00]
```

Comment:
For these alias tests it makes sense to me to include full output in CHECK lines not just the hex values. That way we can see the other name.

Reply:
Agreed. Done. I will go back and fix the existing tests separately.